plugin/build.gradle.kts

```diff
@@ -20,16 +21,19 @@ repositories {
 
 dependencies {
     compileOnly(gradleApi())
-    compileOnly(kotlin("gradle-plugin", "1.2.21"))
-    compileOnly("com.android.tools.build:gradle:3.0.0")
-    compileOnly("org.jetbrains.kotlin:kotlin-native-gradle-plugin:0.6")
-    compile("net.swiftzer.semver:semver:1.0.0")
+    implementation(kotlin("gradle-plugin", "1.2.21"))
+    implementation("com.android.tools.build:gradle:3.0.0")
+    implementation("org.jetbrains.kotlin:kotlin-native-gradle-plugin:0.6")
+    implementation("net.swiftzer.semver:semver:1.0.0")
 
     /*
      * Do not depend upon the gradle script kotlin plugin API. IE: gradleScriptKotlinApi()
      * It's currently in flux and has binary breaking changes in gradle 4.0
      * https://github.com/JLLeitschuh/ktlint-gradle/issues/9
      */
+
+    testImplementation(gradleTestKit())
+    testImplementation("junit:junit:4.12")
```

Comment:
:o: Just curious why you chose Junit 4 over Junit 5. I've been using Junit 5 for all my newest projects.

Reply:
Habits and the requirement of `TemporaryFolder`.
I didn't want to introduce another home grown equivalent for JUnit 5.

Reply:
Fair enough.

Reply:
Would be nice to introduce JUnit version in `PluginVersions`